package.json

```
@@ -13,6 +13,7 @@
     "react-scripts": "5.0.1",
     "redux": "^4.2.1",
     "sass": "^1.60.0",
+    "uuid": "^9.0.0",
     "web-vitals": "^2.1.4"
   },
```

Can we define project dependency explicitly ?

```suggestion
  },
    "engines": {
    "node": "^x.x.x",
    "yarn": "^x.x.x"
  },
```